projects/schematics/src/shared/constants.ts

```diff
@@ -639,3 +639,31 @@ export const CSS_V2_DOCS_URL =
 export const CSS_V3_DOCS_URL =
   'https://sap.github.io/spartacus-docs/css-changes-in-version-3/';
 /***** Storefinder end *****/
+
+/***** Product configurator start *****/
+export const CLI_PRODUCT_CONFIGURATOR_FEATURE = 'ProductConfigurator';
```

Comment:
Keep the constant related to this lib in it's own schematics source code.

Previously we kept that here, because it was reused for migrations, but for new library it doesn't make any sense to keep it here.

Reply:
Removed from schematics